jdaviz/configs/default/plugins/markers/markers.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <j-tray-plugin
-    description='Create and export markers.  Press "m" with the cursor over a viewer to log the mouseover information.  To change the selected layer, click the layer cycler in the mouseover information section of the app-level toolbar.'
+    :description="docs_description || 'Create and export markers.  Press "m" with the cursor over a viewer to log the mouseover information.  To change the selected layer, click the layer cycler in the mouseover information section of the app-level toolbar.'"

```

Comment:
BTW this broke the plugin but I fixed it directly on main via https://github.com/spacetelescope/jdaviz/commit/b58a5a44e7f8df42c02e519194d6967213ec125f . I was in the middle of something else and didn't feel like the overhead of JIRA + new PR.

Reply:
Woops, that one slipped by, thanks for the quick patch